### ISSUE TITLE
Add aria labels to the buttons

### ISSUE THREAD

**Issue by ewmcelvaney** [CodefortheCarolinas/tax-benefit-website#10]
Our buttons should have aria labels to enhance accessibility. To learn more about aria labels, you can look at MDN's page about them: [https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

**Comment by scastoro** [comment]
I can help out with that. Are you referring to the bottoms on the bottom of the page (Outreach guide, sample messages, contact us, etc..)?

**Comment by ewmcelvaney** [comment]
Thanks! I do
